docs/v1/use-cases.md

```diff
@@ -17,37 +17,83 @@ implement.
 a wide variety of SaaS products to users of Kubernetes.
 
-### Exposing existing services outside the cluster
+### Exposing Kubernetes services outside the cluster
 
 It should be possible for service operators whose services are deployed in a
 Kubernetes cluster to publish their services using a service broker.  This
 would allow these operators to participate in the existing service broker
 ecosystem and grow their user base accordingly.
 
-## Low-Level Use Cases
+## CF Service Broker `v2` API Use Cases
 
-These are lower-level use cases the service catalog will implement in service
-to the high-level use cases.
+Initially, the catalog should support the current [CF Service Broker
+API](https://docs.cloudfoundry.org/services/api.html) These are the use cases
+that the service catalog has to implement in order to use that API.
 
-1.  As a service broker operator, I want to be able to register my broker with
+### Managing service brokers
+
+1.  As a catalog operator, I want a user to be able to register a broker with 
     the Kubernetes service catalog, so that the catalog is aware of the services
-    my broker offers
+    that broker offers
+2.  As a catalog operator, I want a user to be able to update a registered
+    broker so that the catalog can maintain the most recent versions of services
+    that broker offers
+3.  As a catalog operator, I want to be able to delete a broker from the 
+    catalog, so that I can keep the catalog clean of brokers I no longer want to
+    support
+
+#### Registering a service broker with the catalog
+
+An admin must register each service broker with the service catalog to
+advertise the services it offers to the catalog.  After the broker has been
+registered with the catalog, the catalog makes a call to the service broker's
+`/v2/catalog` endpoint.  The broker's returns a list of services offered by
+that broker.  Each Service has a set of plans that differentiate the tiers of
+that service.
+
+#### Updating a service broker
+
+Broker authors make changes to the services their brokers offer.  To refresh the
+services a broker offers, the catalog should re-list the `/v2/catalog` endpoint.
+The catalog should apply the result of re-listing the broker to its internal
+representation of that broker's services:
+
+1.  New service present in the re-list results are added
+2.  Existing services are updated if a diff is present
+3.  Existing services missing from the re-list are deleted
+
+TODO: spell out various update scenarios and how they affect end-users
+
+#### Delete a service broker
+
+There must be a way to delete brokers from the catalog.  In Cloud Foundry, it is
+possible to delete a broker and leave orphaned service instances.  We should
+evaluate where all broker deletes should:
+
+1.  Cascade down to the service instances for the broker
+2.  Leave orphaned service instances in the catalog
+3.  Fail if service instances still exist for the broker
+
+## Service backend APIs
```

Comment:
I didn't want to use `versioning the API` or a variant of that because that is a different, very specific thing in my mind... what about 'Versioning backend API' ?